human-dynamics-estimator/src/HumanDynamicsEstimator.cpp

```diff
@@ -23,35 +34,132 @@ double HumanDynamicsEstimator::getPeriod()
+    }    
+    
+    /*
+     * ------Human model loading 
+     */  
+    // TODO: substitution with thrift
+    std::vector<std::string> joints;
+    joints.push_back("jL5S1_rotx");
+    joints.push_back("jL5S1_roty");
+    joints.push_back("jL5S1_rotz");
+    joints.push_back("jL4L3_rotx");
+    joints.push_back("jL4L3_roty");
+    joints.push_back("jL4L3_rotz");
+    joints.push_back("jL1T12_rotx");
+    joints.push_back("jL1T12_roty");
+    joints.push_back("jL1T12_rotz");
+    joints.push_back("jT9T8_rotx");
+    joints.push_back("jT9T8_roty");
+    joints.push_back("jT9T8_rotz");
+    joints.push_back("jT1C7_rotx");
+    joints.push_back("jT1C7_roty");
+    joints.push_back("jT1C7_rotz");
+    joints.push_back("jC1Head_rotx");
+    joints.push_back("jC1Head_roty");
+    joints.push_back("jC1Head_rotz");
+    joints.push_back("jRightC7Shoulder_rotx");
+    joints.push_back("jRightC7Shoulder_roty");
+    joints.push_back("jRightC7Shoulder_rotz");
+    joints.push_back("jRightShoulder_rotx");
+    joints.push_back("jRightShoulder_roty");
+    joints.push_back("jRightShoulder_rotz");
+    joints.push_back("jRightElbow_rotx");
+    joints.push_back("jRightElbow_roty");
+    joints.push_back("jRightElbow_rotz");
+    joints.push_back("jRightWrist_rotx");
+    joints.push_back("jRightWrist_roty");
+    joints.push_back("jRightWrist_rotz");
+    joints.push_back("jLeftC7Shoulder_rotx");
+    joints.push_back("jLeftC7Shoulder_roty");
+    joints.push_back("jLeftC7Shoulder_rotz");
+    joints.push_back("jLeftShoulder_rotx");
+    joints.push_back("jLeftShoulder_roty");
+    joints.push_back("jLeftShoulder_rotz");
+    joints.push_back("jLeftElbow_rotx");
+    joints.push_back("jLeftElbow_roty");
+    joints.push_back("jLeftElbow_rotz");
+    joints.push_back("jLeftWrist_rotx");
+    joints.push_back("jLeftWrist_roty");
+    joints.push_back("jLeftWrist_rotz");
+    joints.push_back("jRightHip_rotx");
+    joints.push_back("jRightHip_roty");
+    joints.push_back("jRightHip_rotz");
+    joints.push_back("jRightKnee_rotx");
+    joints.push_back("jRightKnee_roty");
+    joints.push_back("jRightKnee_rotz");
+    joints.push_back("jRightAnkle_rotx");
+    joints.push_back("jRightAnkle_roty");
+    joints.push_back("jRightAnkle_rotz");
+    joints.push_back("jRightBallFoot_rotx");
+    joints.push_back("jRightBallFoot_roty");
+    joints.push_back("jRightBallFoot_rotz");
+    joints.push_back("jLeftHip_rotx");
+    joints.push_back("jLeftHip_roty");
+    joints.push_back("jLeftHip_rotz");
+    joints.push_back("jLeftKnee_rotx");
+    joints.push_back("jLeftKnee_roty");
+    joints.push_back("jLeftKnee_rotz");
+    joints.push_back("jLeftAnkle_rotx");
+    joints.push_back("jLeftAnkle_roty");
+    joints.push_back("jLeftAnkle_rotz");
+    joints.push_back("jLeftBallFoot_rotx");
+    joints.push_back("jLeftBallFoot_roty");
+    joints.push_back("jLeftBallFoot_rotz");
+    
+    
+    std::string modelFilename = rf.find("humanModel").asString();
```

Comment:
I am not sure if this is enough if the file name is not an absolute file name.
Probably you have to find the file before passing the full path to the `ModelLoader`